src/assets/styles/10-settings/_sizes.scss

```diff
@@ -15,5 +15,5 @@
 
   --secondary-nav-height: calc(2 * var(--space-unit));
 
-  --search-height-md: calc(8 * --var(--space-unit));
+  --search-height-md: calc(8 * var(--space-unit));
```

Comment:
Sorry - this was a bug I found and fixed when I was looking at upgrading postcss-css-variables... felt unnecessary to remove again!